en/index.html

```diff
@@ -265,7 +265,7 @@
     <div class="why-us-box-shadow">
       <div class="why-us-flexbox">
         <div class="why-us-group">
-          <img class="why-us-icon" src="/assets/mark/easy.svg" />
+          <img class="why-us-icon" src="/assets/mark/easy.svg" aria-label="easy icon" />
```

Comment:
From MDN[1]:
> ... aria-label is intended for use on interactive elements ...

These images aren't interactive (if I understand correctly). Do we want alt text here instead? Or maybe should these be aria-labelledby the following headings?

[1]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

Reply:
@ras0219-msft, I have incorporated review comments. Could you please review and trigger the build on these changes.